relayer/go.mod

```diff
@@ -3,27 +3,30 @@ module github.com/snowfork/polkadot-ethereum/relayer
 go 1.13
 
 require (
-	github.com/aristanetworks/goarista v0.0.0-20201012165903-2cb20defcd66 // indirect
-	github.com/btcsuite/btcd v0.20.1-beta // indirect
+	github.com/StackExchange/wmi v0.0.0-20190523213315-cbe66965904d // indirect
+	github.com/allegro/bigcache v1.2.1 // indirect
+	github.com/aristanetworks/goarista v0.0.0-20201022192228-4e6fdcf7f221 // indirect
+	github.com/btcsuite/btcd v0.21.0-beta // indirect
 	github.com/centrifuge/go-substrate-rpc-client v2.0.0-rc6+incompatible
-	github.com/deckarep/golang-set v1.7.1 // indirect
-	github.com/ethereum/go-ethereum v1.8.23
-	github.com/fatih/color v1.10.0 // indirect
+	github.com/ethereum/go-ethereum v1.9.24
```

Comment:
This is the reason for the compilation issue. Ethashproof needs to be upgraded to 1.9 before you can use 1.9 here. Golang uses one major version of a package for a build because, by semvar standards, 1.9 should be backwards compatible with 1.8